client/app.js

```diff
@@ -17,9 +17,42 @@ angular.module('ripeT', ['ngMap'])
 })
 
 .controller('MainController', function ($scope, $location, Factory) {
-  $scope.getNumber = function(num) {
-    console.log(num, Array(Math.round(num)));
```

Comment:
If you want to leave this console log for debugging purposes, you might want to give it a more descripting message